react/components/FacetItem.js

```diff
@@ -21,6 +21,8 @@ const FacetItem = ({ facet }) => {
   const { query, map } = useContext(QueryContext)
 
   const handleChange = () => {
+    const urlParams = new URLSearchParams(window.location.search)
```

Comment:
could this ever be run on SSR? if so, shouldn't you handle the case where `window.location` doesn't exist?

Reply:
yes! because this is on the event handler